src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -996,4 +1002,53 @@ private String boxAuthorization(HttpServletRequest request,
 			return "Authorized";
 		}
 	}
+
+	/**
+	 * upload resource files into Box folder
```

Comment:
The code looks fine, but it doesn't seem consistent with this comment - this method seems to be uploading a list of sites that are candidates for bulk migration (and eventually bulk migration of emails archive as well as resource files). I'd suggest changing the method description, and also there probably should be a "TBD"  comment somewhere since the bulkUpliadSiteIds are not currently used anywhere afaict